index.js

```diff
@@ -1 +1,108 @@
 // implement your API here
+const express = require("express");
+const cors = require("cors");
+
+const { find, findById, insert, update, remove } = require("./data/db");
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
```

Comment:
I think cors middleware should precede express.json middleware

Reply:
It should be after according to the readme instructions